solutions/client-setup/folder.yaml

```diff
@@ -18,6 +18,6 @@ metadata:
   name: clients.client-name # kpt-set: clients.${client-name}
   namespace: hierarchy
 spec:
-  displayName: client-name # kpt-set: ${client-name}-${environment}
```

Comment:
@cartyc @fmichaelobrien  would that impact the functionality if multiple lz are deployed within a single organization ?

Reply:
Should not, the folder id is distinct, there can be identical names in the leaf nodes of adjacent folder branches